jdaviz/configs/specviz/plugins/viewers.py

```diff
@@ -303,16 +303,17 @@ def add_data(self, data, color=None, alpha=None, **layer_state):
 
         Parameters
         ----------
-        spectrum : :class:`glue.core.Data`
+        spectrum : :class:`glue.core.data.Data`
             Data object with the spectrum.
-        color: color specification
```

Comment:
Unless someone can tell me exactly what Python data types "color specification" refers to, we should just leave it blank.